slave/process-scim/lib/process-scim.example

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+'''
+EXAMPLE of python script for processing SCIM files from Perun
+'''
+
+USERS_FILE = '/var/spool/perun/scim-service-users.scim'
+GROUPS_FILE = '/var/spool/perun/scim-service-grops.scim'
+
+import os
+import sys
+import json
+
+''' GETTING DATA FROM JSON '''
+json_users = open(USERS_FILE)
+users_data = json.load(json_users)
+json_users.close()
+
+json_groups = open(GROUPS_FILE)
+groups_data = json.load(json_groups)
+json_groups.close()
+
+''' PARSING DATA FROM users.scim '''
+for item in users_data:
+	userJSON_ids.append(int(item['id']))	
+	tmpUser = User()
```

Comment:
Pyhton code style recommends using underscores instead of cammelCase. I don't mind either, but seeing combination of both in one file really draws ones attention :-)

Reply:
Fixed.